src/tests/ftest/util/general_utils.py

```diff
@@ -538,3 +538,16 @@ def get_log_file(name):
 
     """
     return os.path.join(os.environ.get("DAOS_TEST_LOG_DIR", "/tmp"), name)
+
+
+def check_uuid_format(uuid):
+    """Checks a correct UUID format.
+
+    Args:
+        uuid (str): Pool or Conatiner UUID.
```

Comment:
(style)  'Conatiner' may be misspelled - perhaps 'Container'?

Reply:
Done